intera_examples/scripts/camera_display.py

```diff
@@ -62,30 +69,46 @@ def main():
         '-c', '--camera', type=str, default="head_camera",
         choices=valid_cameras, help='Setup Camera Name for Camera Display')
     parser.add_argument(
-        '-r', '--raw', action='store_true', 
+        '-r', '--raw', action='store_true',
         help='Specify use of the raw image (unrectified) topic')
     parser.add_argument(
         '-e', '--edge', action='store_true',
         help='Streaming the Canny edge detection image')
+    parser.add_argument(
+        '-g', '--gain', type=int,
+        help='Set gain for camera (-1 = auto)')
+    parser.add_argument(
+        '-x', '--exposure', type=float,
+        help='Set exposure for camera (-1 = auto)')
     args = parser.parse_args()
 
     print("Initializing node... ")
     rospy.init_node('camera_display', anonymous=True)
     try:
-        camera = intera_interface.Cameras()
+        cameras = intera_interface.Cameras()
     except OSError as e:
         rospy.logfatal("Could not find all of the expected cameras for this robot.\n"
                 "Please contact Rethink support: support@rethinkrobotics.com")
         return
-    if not camera.verify_camera_exists(args.camera):
+    if not cameras.verify_camera_exists(args.camera):
         rospy.logerr("Invalid camera name, exiting the example.")
         return
-    camera.start_streaming(args.camera)
+    rospy.loginfo("Opening camera '{0}'...".format(args.camera))
+    cameras.start_streaming(args.camera)
     rectify_image = not args.raw
     use_canny_edge = args.edge
-    camera.set_callback(args.camera, show_image_callback,
+    cameras.set_callback(args.camera, show_image_callback,
         rectify_image=rectify_image, callback_args=(use_canny_edge, args.camera))
 
+    # optionally set gain and exposure parameters
```

Comment:
Should `gain` and `exposure` be set before streaming & callbacks are started?

Reply:
It shouldn't really matter - and starting streaming + the window coming up takes a while anyways, so might as well start that early.